guides/common/modules/proc_disable-transparent-hugepage.adoc

```diff
@@ -0,0 +1,11 @@
+[id="Disable_Transparent_Hugepage_{context}"]
+= Disable Transparent Hugepage
+
+Transparent Hugepage is a memory management technique used by the Linux kernel to reduce the overhead of using the Translation Lookaside Buffer (TLB) by using larger sized memory pages.
+Due to databases having Sparse Memory Access patterns instead of Contiguous Memory access patterns, database workloads often perform poorly when Transparent Hugepage is enabled.
+To improve PostgreSQL or Redis performance, disable Transparent Hugepage.
+In deployments where the databases are running on a separate server(s), there may be a small benefit to using Transparent Hugepage on the {ProjectServer} only.
```

Comment:
```suggestion
In deployments where the databases are running on separate servers, there may be a small benefit to using Transparent Hugepage on the {ProjectServer} only.
```

According to IBM style guide, we should not add "(s)" to pluralize words or "s" for abbreviations.